rundeckapp/grails-app/controllers/rundeckapp/UrlMappings.groovy

```diff
@@ -145,6 +145,7 @@ class UrlMappings {
                 action: 'apiProjectExportAsyncDownload'
         )
         "/api/$api_version/project/$project/import"(controller: 'project',action: 'apiProjectImport')
+        "/api/$api_version/project/$project/async/import-status"(controller: 'project',action: 'apiProjectAsyncImportStatus')
```

Comment:
The API url path should follow the same pattern as the async export status.  `GET /api/19/project/[PROJECT]/export/status/[TOKEN]`